giskard/ml_worker/websocket/listener.py

```diff
@@ -100,6 +102,8 @@ async def handle_result(future: Union[Future, Callable[..., websocket.WorkerRepl
                 error_str=e.info.message, error_type=e.info.type, detail=e.info.stack_trace
             )
             logger.warning(e)
+        except TimeoutError as e:
```

Comment:
When is this happening vs the cancelled one ?

Reply:
indeed, I tested, the timeout exceptions result in `CancelledError` so I don't see the need for a separate TimeoutError handler